.env.example

```diff
@@ -29,6 +29,9 @@ CL_API_URLS=https://<consensus-layer-api-url>
 # Validator registry source will be "lido" or "file" (optional).
 VALIDATOR_REGISTRY_SOURCE=lido
 
+# Effective Balance is 32 by default.
+# EFFECTIVE_BALANCE=32
```

Comment:
Thank you for your suggestion!

I would say it is
```suggestion
# EFFECTIVE_BALANCE_INCREMENTS=32
```

Reply:
@vgorkavenko thanks for reviewing this PR and I agree with that.
The new commit 93d6c15df33ec0959607f115912f22e231cefaed pushed for applying your comment :)